browser/mapping.go

```diff
@@ -280,8 +281,22 @@ func mapElementHandle(vu moduleVU, eh *common.ElementHandle) mapping {
 			return mapFrame(vu, f), nil
 		},
 		"press": eh.Press,
-		"screenshot": func(opts goja.Value) goja.ArrayBuffer {
-			return eh.Screenshot(opts, vu.LocalFilePersister)
+		"screenshot": func(opts goja.Value) (*goja.ArrayBuffer, error) {
+			ctx := vu.Context()
+
+			popts := common.NewElementHandleScreenshotOptions(eh.Timeout())
+			if err := popts.Parse(ctx, opts); err != nil {
+				return nil, fmt.Errorf("parsing frame screenshot options: %w", err)
+			}
+
+			bb, err := eh.Screenshot(popts, vu.LocalFilePersister)
+			if err != nil {
+				return &goja.ArrayBuffer{}, err //nolint:wrapcheck
```

Comment:
What happens when return `nil` here?

```suggestion
				return nil, err //nolint:wrapcheck
```

Reply:
Resolved in b7dff9f3c73b0dd207539e56bcb01ab47cf0911e